run/authentication/auth.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START run_service_to_service_auth]
```

Comment:
Please see the Node.js Policies on the DPE team page to up the placement of the region tags and imports.

Reply:
I've wrapped the sample in a function, if there's more to do to correctly place the region tags & imports can you let me know? I've followed [this guide](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md#canonical-samples)

Reply:
Good. We use the region tags so our docs know what code to grab. For node.js we don't care about the function signature, but we do care about the imports. We don't need a function within a function. You can just make the outer function async and even though the example uses "main" let's use a better name.